### DISCUSSION TITLE
Future Laravel support?

### DISCUSSION THREAD

**Discussion by ZED-Magdy** [doctrine/doctrine2#9215] [Support Questions] [answered]
Is there any plans to make a Laravel support in the future?
I use both Symfony and Laravel, I don't like Eloquent IMHO Doctrine is much better,
The Laravel-Doctrine package is out of date and I think it's not supported any more since its last release is from 2020,
I'd love using Doctrine in Laravel.

**Comment by beberlei** [comment, marked as answer]
No plans from our end, this would be for a third party to implement.
